detect_analysis.R

```diff
@@ -0,0 +1,47 @@
+---
+title: "Reporting systems sub-analysis"
+author: "Emily Linebarger"
+date: "2/17/2021"
+output: pdf_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = FALSE)
+library(data.table)
+
+dt = fread("C:/Users/eklin/Documents/repos/data_557/prepped_data/sixmonth.csv")
+```
+
+First, run a simple scatter to see what the correlation between the overall score and the "early detection" sub-component looks like. 
+```{r exploratory_analysis}

```

Comment:
Is this an Rmd file or an R file? It looks like Rmd, but the extension is R.

Reply:
This one was a mistake - I'll delete.